src/Common/ZooKeeper/ZooKeeperImpl.h

```diff
@@ -2,6 +2,7 @@
 
 #include <base/defines.h>
 #include <base/types.h>
+#include "Common/Histogram.h"
```

Comment:
Please don't add includes to headers unless they are absolutely necessary. Instead add them only to the implementation files. And please use `<>`. We normally reserve `"` for autogenerated files.

Reply:
done

Reply:
This is still there